libs/components/forms/src/lib/modules/checkbox/checkbox.component.ts

```diff
@@ -304,9 +304,7 @@ export class SkyCheckboxComponent implements ControlValueAccessor, OnInit {
   #controlValueAccessorChangeFn: (value: any) => void = (value) => {};
 
   #emitChangeEvent(): void {
-    const event = new SkyCheckboxChange();
-    event.source = this;
-    event.checked = this.#_checked;
+    const event: SkyCheckboxChange = { source: this, checked: this.#_checked };
```

Comment:
Change `this.#_checked` to use `this.checked` to avoid our rule of not referencing a `#_` property outside of its corresponding getter/setter.